libcontainer/cgroups/fs/cpuacct.go

```diff
@@ -20,7 +20,7 @@ const (
 	nanosecondsInSecond = 1000000000
 )
 
-var clockTicks = uint64(system.GetClockTicks())
+var clockTicks = system.GetClockTicks()
```

Comment:
I'd rather drop `system.GetClockTicks()` entirely and have this as
```go
const clockTicks = 100
```

(with a comment that explains where it is coming from)

Reply:
Having it as const will help compiler to optimize the calculations, and I doubt there are any external users of `system.GetClockTicks()`.

Reply:
It may still be used in some locations outside of runc, so we should probably be careful with removing `system.GetClockTicks()` altogether, but do you want me to put a const in this file (so that it doesn't depend on `libnetwork/system` ?

Reply:
Go API compatibility is not guaranteed across releases (at least until v1.0.0 GA), so we can safely remove `system.GetClockTicks()`.